crowdin.yml

```diff
@@ -0,0 +1,7 @@
+files:
+  - source: /lang/languages.json
+    translation: /lang/languages-%two_letters_code%.json
+  - source: /lang/main.json
+    translation: /lang/main-%two_letters_code%.json
+  - source: /android/**/values/strings.xml
```

Comment:
Can you remove this, this does not need to be translated at this is just the name of the app and nothing more.

Reply:
Sure, I'll remove XML files from the config